frontend/packages/console-dynamic-plugin-sdk/README.md

```diff
@@ -31,6 +31,37 @@ provided by Console to its dynamic plugins.
 _[OpenShift Console Dynamic Plugins feature page][console-doc-feature-page]_ - A high-level overview of
 dynamic plugins in relation to OLM operators and cluster administration.
 
+## Plugin project references
+
+If you're new to dynamic plugins, we suggest to clone the [Console plugin template][console-plugin-template]
+repo and follow its instructions on setting up a local plugin development environment. We recommend running
+the Console application via container image to avoid having to build Console locally.
```

Comment:
```suggestion
the Console application with a container image to avoid having to build Console locally.
```
Just a suggestion, we avoid using via in the docs.